dags/oup/parser.py

```diff
@@ -56,6 +56,11 @@ def __init__(self) -> None:
                 source="front/article-meta/title-group/article-title",
                 all_content_between_tags=True,
             ),
+            CustomExtractor(
+                destination="journal_title",
+                extraction_function=self._get_journal_title,
+                required=True,
```

Comment:
required is also default prop, right?

Reply:
If you are talking about "required" in custom extractor it is False by default